app/controllers/keywords_controller.rb

```diff
@@ -43,4 +44,8 @@ def save_keywords
   def keywords_form
     @keywords_form ||= KeywordsForm.new(current_user)
   end
+
+  def keywords_query
+    @keywords_query ||= KeywordsQuery.new(current_user.keywords, params)
```

Comment:
How about using strong params to permit only supported params? https://api.rubyonrails.org/v6.1.4/classes/ActionController/Parameters.html#method-i-permit

Reply:
Good suggestion! Learn something new! 🥰
Fixed in [`3ee058b` (#48)](https://github.com/suho/google-search-ruby/pull/48/commits/3ee058b905d63d10c5600c642c1a482b7daccc70)